proc_fusion_spell.lua

```diff
@@ -255,20 +284,99 @@ function (fusfilter,matfilter,extrafil,extraop,gc2,stage2,exactcount,value,locat
 					if sel[1]==e then
 						Fusion.CheckAdditional=checkAddition
 						Fusion.ExtraGroup=extragroup
+						--"Cyberdark Chimera" OPT EFFECT_EXTRA_FUSION_MATERIAL check
+						--Same checks etc as line 119.
+						local extra_feff_mg=mg1:Filter(Card.IsHasEffect,nil,EFFECT_EXTRA_FUSION_MATERIAL)
+						if #extra_feff_mg>0 then
+							local extra_feff=extra_feff_mg:GetFirst():IsHasEffect(EFFECT_EXTRA_FUSION_MATERIAL)
+							if extra_feff and Duel.GetFlagEffect(tp,extra_feff:GetHandler():GetCode())>0 then
```

Comment:
It could be easier to directly use Effect.CheckCountLimit and Effect.UseCountLimit afterwards when applying it